includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -303,13 +303,7 @@ function ( $id ) use ( $is_test_mode ) {
 	 * @return bool
 	 */
 	public static function is_enabled() {
-		$stripe_settings     = WC_Stripe_Helper::get_stripe_settings();
-		$connection_type_key = WC_Stripe_Mode::is_test() ? 'test_connection_type' : 'connection_type';
-
-		// If the account is not a Connect OAuth account, we can't use the payment method configurations API.
-		if ( ! isset( $stripe_settings[ $connection_type_key ] ) || 'connect' !== $stripe_settings[ $connection_type_key ] ) {
-			return false;
-		}
```

Comment:
I think we should still check for known invalid values.
```suggestion
		$connection_type_key = WC_Stripe_Mode::is_test() ? 'test_connection_type' : 'connection_type';

		// If the account is known to be something other than a Connect OAuth account, we can't use the payment method configurations API.
		if ( isset( $stripe_settings[ $connection_type_key ] ) && 'connect' !== $stripe_settings[ $connection_type_key ] ) {
			return false;
		}

Reply:
It won't let me commit changes to deleted lines, but suggestion pushed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4581/commits/5b0cf289cbdbfb4acb7e47ae5d0e8780d478a58a

Reply:
During the team call, I remember @diegocurbelo saying we should just remove `connection_type` checks. Was that just for the other PR, or should we remove it from this as well?

Reply:
I think we should remove the `connection_type` check in this PR